sql_utils.py

```diff
@@ -46,61 +46,61 @@ def set_up_sqlite():
     cop_install_podnet_b_warn           = 0b000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001100000000110000000000000000001
-    cop_install_podnet_b_fail           = 0b000000000000000000000000000000000000000000000000000000000000000000000000000000000000001110000101111000000000001100001110
+    cop_install_podnet_b_fail           = 0b000000000000000000000000000000000000000000000000000000000000000000000000001111111111101110000101111000000000001100001110
     cop_install_appliance_a_warn        = 0b000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000110000000000000000001
-    cop_install_appliance_a_fail        = 0b000000000000000000000000000000000000000000000000000000000000000000000000000000000000001110000101111000000000001100010110
+    cop_install_appliance_a_fail        = 0b000000000000000000000000000000000000000000000000000000000000000000000000001100110000001110000101111000000000001100010110
     cop_reinstall_podnet_a_warn         = 0b000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001100000000110000000000000000001
-    cop_reinstall_podnet_a_fail         = 0b000000000000000000000000000000000000000000000000000000000000000000000000000000000000001110000101111000000000001100001110
+    cop_reinstall_podnet_a_fail         = 0b000000000000000000000000000000000000000000000000000000000000000000000000001111111111101110000101111000000000001100001110
     region_validate_podnet_a_warn       = 0b000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001100000000000000000000000000001
-    region_validate_podnet_a_fail       = 0b000000000000000000000000000000000000000000000000000000000000000000000000000000000000001110011010111011111111111111101110
+    region_validate_podnet_a_fail       = 0b000000000000000000000000000000000000000000000000000000000000000000000000001111111111101110011010111011111111111111101110
     region_validate_podnet_b_warn       = 0b000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001100000000110000000000000000001
-    region_validate_podnet_b_fail       = 0b000000000000000000000000000000000000000000000000000000000000000000000000000000000000001110011010111011111111111111101110
+    region_validate_podnet_b_fail       = 0b000000000000000000000000000000000000000000000000000000000000000000000000001111111111101110011010111011111111111111101110
     region_validate_appliance_a_warn    = 0b000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000110000000000000000001
-    region_validate_appliance_a_fail    = 0b000000000000000000000000000000000000000000000000000000000000000000000000000000000000011110011010111000000000011100010110
+    region_validate_appliance_a_fail    = 0b000000000000000000000000000000000000000000000000000000000000000000000000001100110000011110011010111000000000011100010110
```

Comment:
```suggestion
    region_validate_appliance_a_fail    = 0b000000000000000000000000000000000000000000000000000000000000000000000000001100110000111110011010111000000000011100010110
```

As `ipv4_link_cpe` (35) is in the appliance json for a region, it should be tested

Reply:
ipv4_link_cpe (35), tests whether it is in ipv4_link_subnet, so we need `ipv4_link_subnet` in config.json in appliance?

FYI: The need of `ipv4_link_cpe` is for VPNs by Robot(Primitive).

Reply:
There should be two tests then for the pe and cpe addresses

1. To ensure it is in the config.json and is an IP Address
2. To ensure it is in the relevant link subnet

That way point one can be tested for appliances with the region flavour and point two ignored.